app/build.gradle

```diff
@@ -35,9 +37,10 @@ android {
     }
     buildFeatures {
         compose true
+        viewBinding true
```

Comment:
this is probably not needed in a compose app

Reply:
Done